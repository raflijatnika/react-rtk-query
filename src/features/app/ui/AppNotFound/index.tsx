const AppNotFound = () => {
  return <h1>Not Found</h1>
}

export { AppNotFound }
