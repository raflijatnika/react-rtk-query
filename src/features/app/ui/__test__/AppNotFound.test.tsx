// Test
import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'

// UI
import { AppNotFound } from '../AppNotFound'

it('AppNotFound should render', () => {
  render(<AppNotFound />)
})

it('AppNotFound should have snapshot', () => {
  const tree = renderer.create(<AppNotFound />).toJSON()

  expect(tree).toMatchSnapshot()
})

it('AppNotFound should have Not Found', () => {
  render(<AppNotFound />)
  expect(screen.getByText('Not Found')).toBeInTheDocument()
})
