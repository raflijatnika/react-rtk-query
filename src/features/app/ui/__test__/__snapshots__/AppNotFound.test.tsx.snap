// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNotFound should have snapshot 1`] = `
<h1>
  Not Found
</h1>
`;
